src/app/public/matchers/matchers.ts

```diff
@@ -323,12 +411,40 @@ interface SkyNormalMatchers<T> extends jasmine.Matchers<T> {
    * @param trimWhitespace [true] Whether or not to trim whitespace from the actual element text before comparison.
    */
   toHaveText(expectedText: string, trimWhitespace?: boolean): void;
-}
 
-/**
- * Interface for custom Sky matchers (includes original jasmine matchers).
- */
-export interface SkyMatchers<T> extends SkyNormalMatchers<T>, SkyAsyncMatchers<T> {}
+  /**
+   * `expect` the actual component to be accessible based on Web Content Accessibility
+   * Guidelines 2.0 (WCAG20) Level A and AA success criteria.
+   * @deprecated Use `expectAsync(element).toBeAccessible()` instead.
```

Comment:
Should this example also include an `await` the proceeds the `expectAsync` ?

Reply:
Same goes for the other methods below.

Reply:
Done.